pandas/io/json/_table_schema.py

```diff
@@ -96,8 +96,9 @@ def set_default_names(data):
         if len(nms) == 1 and data.index.name == "index":
             warnings.warn("Index name of 'index' is not round-trippable.")
         elif len(nms) > 1 and any(x.startswith("level_") for x in nms):
-            warnings.warn("Index names beginning with 'level_' are not "
-                          "round-trippable.")
+            warnings.warn(
+                "Index names beginning with 'level_' are not " "round-trippable."
+            )
```

Comment:
There were two " in the original because the string was split in two lines. 
The new string should be `"Index names beginning with 'level_' are not round-trippable."` (remove the `" "` from the middle of string).